packages/core/src/types.ts

```diff
@@ -217,6 +217,15 @@ export type Histogram = BasePlot & {
 
 export type Plot = Scatterplot | Histogram;
 
+// eslint-disable-next-line no-shadow
```

Comment:
Again, I think it's best practice to document eslint disabling. Which var is being shadowed? Why is that unimportant?